src/plonk/lookup.rs

```diff
@@ -25,18 +25,30 @@ impl<F: Field> Argument<F> {
     pub(crate) fn required_degree(&self) -> usize {
         assert_eq!(self.input_expressions.len(), self.table_expressions.len());
 
+        // The first value in the permutation poly should be one.
         // degree 2:
-        // l_0(X) * (1 - z'(X)) = 0
+        // l_0(X) * (1 - z(X)) = 0
```

Comment:
```suggestion
        // l_0(X) * (z(X) - 1) = 0
```
following https://github.com/zcash/orchard/issues/143.

Reply:
Oh, this will require changing the constraint system as it's not just an innocent notational quirk. I prefer `1 - X` tbh, it's more common in our other SNARK code, and we've already implemented everything like this using `Polynomial::one_minus`.